### ISSUE TITLE
getName result is wrong for img with alt and title

### ISSUE THREAD

**Issue by jribbens** [xi/aria-api#10]
If you consider the following HTML:
```html
<img src="..." title="title">
<img src="..." alt="" title="title">
<img src="..." alt="alt" title="title">
```
then the `getName()` results for these images should be `'title'`, `''`, and `'alt'` respectively. However the returned results are `'title'`, `'title'`, and `'alt'`.

This is explicitly defined by this specification: https://www.w3.org/TR/html-aam-1.0/#img-element-accessible-name-computation

**Comment by xi** [comment]
Nice catch!

I pushed a potential fix as #11. Unfortunately this breaks [WPT 566](https://github.com/web-platform-tests/wpt/blob/master/accname/name_test_case_566-manual.html). This might be a result of the unclear relation between accname and HTML-AAM (see https://github.com/w3c/accname/issues/8).

I don't really have time to get into this right now. @jribbens do you think you could clarify this upstream?

**Comment by xi** [comment]
Fix is published in 0.4.6
